de/The-Manual/Beginning-With-CakePHP/Where-to-Get-Help.rst

```diff
@@ -49,7 +49,7 @@ bring Deinen Sturzhelm mit.
 CakeForge
 =========
 
-`http://www.cakeforge.org <http://www.cakeforge.org>`_
+`https://www.cakeforge.org <https://www.cakeforge.org>`_
```

Comment:
This appears to link to an online poker site now. Should we remove it? @markstory

Reply:
Yeah we don't need to send links to poker sites.

Reply:
remove